src/test/java/com/github/jscancella/conformance/profile/BagitProfileTest.java

```diff
@@ -0,0 +1,70 @@
+package com.github.jscancella.conformance.profile;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Type;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class BagitProfileTest extends AbstractBagitProfileTest {
+
+  @Test
+  public void testEquals() {
```

Comment:
This seems to be testing a bunch of cases of `.equals()`, can you split them out into separate tests and change the test name to be more meaningful? For example, your first test, tests two value identical objects return true when using equals, so I would make that its own test and name it `testValueIdenticalAreEqual`